src/views/FAQ/FAQ.tsx

```diff
@@ -267,10 +269,11 @@ export const FAQPage: FunctionComponent = () => (
           <FAQItem question="Why isn’t my package displayed on Construct Hub?">
```

Comment:
"Who owns the constructs in Construct Hub?" should be updated with the latest answer